src/bci_build/package/appcontainers.py

```diff
@@ -204,15 +204,23 @@ def _envsubst_pkg_name(os_version: OsVersion) -> str:
     + [(pg_ver, OsVersion.TUMBLEWEED) for pg_ver in (14, 13, 12)]
 ]
 
+
+def _generate_prometheus_family_healthcheck(port: int) -> str:
+    return f"""HEALTHCHECK --interval=5s --timeout=5s --retries=5 \
+    CMD ["/bin/sh", "-c", "[ \"$(/usr/bin/curl http://localhost:{port}/-/healthy)" = \"Healthy\" ]"]
```

Comment:
Unfortunately the API specification is not consistent here. Every component returns a different [response text](https://github.com/SUSE/BCI-tests/pull/503/files#diff-5f219bf2012d07759723f500675e1769613674c398febbb2792a508257126ca1R39). Let's add another parameter to the function.